src/schema/managing-code.ts

```diff
@@ -0,0 +1,12 @@
+import { char, pgTable } from "drizzle-orm/pg-core";
+import { z } from "zod";
+
+export const managingCode = pgTable("managing_code", {
+  id: char("id", { length: 3 }).primaryKey(),
+});
+
+export const managingCodeEntitySchema = z.object({
+  id: z.string().regex(RegExp("^[A-Z][0-9]{3}")),
```

Comment:
Nit: This regex is something to keep an eye on. I'm realizing now that I never documented the expected shape of the managing codes. All of the characters are expected to be numeric and the regex could be more restrictive to reflect this: ```^[0-9]{3}```

But we should also question whether we want to use a regex at all. zod provides a .length method that could also meet our needs and be more performant than a regex: ```z.string().length(3)```